### PR TITLE
ci: use cargo-audit without a wrapper Action

### PR DESCRIPTION
**Pull request by eugene-babichenko**

### CODE REVIEW

File: .github/workflows/audit.yml [hunk 1]
```diff
@@ -33,7 +33,8 @@ jobs:
           restore-keys: |
             cargo-index-
 
-      # FIXME switch back to actions-rs/audit-check when the fix is merged
-      - uses: eugene-babichenko/audit-check@dont-override-cargo-lock
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install cargo audit
+        run: cargo install cargo-audit
```
Comment (mzabaluev) [inline .github/workflows/audit.yml:37]: This takes a long time to build.

Can we supply a docker container, for example, reusing our old [CI Dockerfile](https://github.com/input-output-hk/rust-ci-docker/blob/master/stable/Dockerfile) which used to also have cargo audit?

Comment (eugene-babichenko) [reply .github/workflows/audit.yml:37]: Yeah I think we can. What is the difference between `circleci/rust` and `rust` images?

Comment (Zeegomo) [reply .github/workflows/audit.yml:37]: `circleci/rust` is a legacy image  (`cimg/rust` is the replacement), but I don't know the difference between that and `rust`.

**Comment by eugene-babichenko** [comment]
Superseded by #3434

**Pull request closed without merging**